### PR TITLE
 Update MySQL appliance and depencies to v17.0.

### PR DESCRIPTION
**Pull request by Christie018**
Update implements bug fix to close PR #1614. Refactoring to lighttpd package and landing page done in makefile and plan/main. Additionally, readme.rst file was reviewed to ensure all information and links are still relevant and working. Update has been tested to ensure full functionality.

### CODE REVIEW

**Review by OnGle — APPROVED** [review]
lgtm :)

File: Makefile [hunk 1]
```diff
@@ -1,8 +1,10 @@
 WEBMIN_FW_TCP_INCOMING = 22 80 443 3306 12320 12321 12322
 
-COMMON_OVERLAYS = adminer tkl-webcp lighttpd
+COMMON_OVERLAYS = adminer tkl-webcp 
 COMMON_CONF = adminer-lighttpd adminer-mysql tkl-webcp
 
+include $(FAB_PATH)/common/mk/turnkey/lighttpd.mk
+include $(FAB_PATH)/common/mk/turnkey/php-fpm.mk
 include $(FAB_PATH)/common/mk/turnkey/php.mk
```
Comment (JedMeister) [inline Makefile:8]: FWIW `php-fpm.mk` includes `php.mk` (see https://github.com/turnkeylinux/common/blob/master/mk/turnkey/php-fpm.mk#L3) so this line can be removed.

**Review by JedMeister — COMMENTED** [review]
Only one very minor point Mattie. Great work! :)

File: Makefile (hunk 1 above)
Comment (Christie018) [reply Makefile]: I went through and changed the code to remove the extra include php.mk file from common.

**Pull request merged**